proc_scripts/plotting.py

```diff
@@ -124,3 +124,52 @@ def plot_curve(fl, f, name, guess=None):
             fl.plot(guess, '-', label='initial guess')
         text(0.75, 0.25, f.latex(), transform=gca().transAxes, size='large',
                 horizontalalignment='center',color='k')
+    def next(self, *arg, **kwargs):
+        kwargs.update({"figsize": (9, 6), "legend": True})
+        super().next(*arg, **kwargs)
```

Comment:
move this to the top of the class definition (this should be the first function that you define)